webserver.py

```diff
@@ -266,9 +309,14 @@ def enqueue(self, youtube_id):
 
 	@cherrypy.expose
 	@cherrypy.tools.json_out()
-	def list(self):
+	def status(self):
 		elems = self.elems()
-		return {"listing": elems, "titles": self.titles(set(elem["ytid"] for elem in elems)), "volume": get_volume()}
+		raw_status = redis.get("musicastatus")
+		playback_status = json.loads(raw_status.decode()) if raw_status else {}
+		playback_status["listing"] = elems
+		playback_status["titles"] = self.titles(set(elem["ytid"] for elem in elems))
+		playback_status["volume"] = get_volume()
+		return playback_status
```

Comment:
It might be good to keep the name of the endpoint the same, for the sake of integrations like asuhl's mz command?